sideboard/lib/_utils.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import unicode_literals
```

Comment:
Nothing in this file is new; it's all just been moved here from ``sideboard/lib/__init__.py``.